docker-compose.yml

```diff
@@ -36,3 +36,15 @@ services:
       interval: 30s
       timeout: 10s
       retries: 3
+  redis:
+    image: redis:5.0-alpine
+    expose:
+      - '6379'
+#    volumes:
+#      - /srv/humhub/redis:/data
+    environment:
+      #- REDIS_PASSWORD=redis_password
+      - ALLOW_EMPTY_PASSWORD=yes
```

Comment:
is there an easy way to test this with psssword? I don't know many password-less redis in production environments and testing this during CI would avoid possible troubles.

Reply:
> is there an easy way to test this with psssword?

@mriedmann on redis side yes, just need to uncomment the `REDIS_PASSWORD`.
On HumHub side, I need to check if it correctly manages Redis connection with a password (config and code supports it but just want to make sure there is no bug).